### ISSUE TITLE
Branch: subpluginSettings, File: settings.php, Line: 92

### ISSUE THREAD

**Issue by Mitch1404** [Peterburnett/QUTCapstone2020#21]
Exception - Call to a member function add() on null

I get this error, its because of the $ADMIN param, I think. Not sure. I am trying to add the sub-plugin settings.

**Comment by harunaka11037** [comment]
I don't know why it's causing errors on your branch, but I tried implementing it myself on a new branch and it works fine.  I'm not sure what the difference is, the only thing that stood out to me was that you forgot to copy over the method `get_settings_section_name`, which is called by `load_settings`.  However, I tried adding this to your branch and it was still giving me the same errors, so that might only be part of the issue.  The only other differences are that you've got two subplugins and your formatting for the versioning in version.php for those subplugins is wrong.
